src/AutoSubscribe/Entity/User.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Clearbooks\LabsMysql\AutoSubscribe\Entity;
+
+use Clearbooks\Labs\AutoSubscribe\Entity\User as IUser;
+
+/**
+ * Created by PhpStorm.
+ * User: Volodymyr
+ * Date: 03/09/2015
+ * Time: 12:16
+ */
+class User implements IUser
```

Comment:
Is there a reason we have so many user classes that all do the exact same things?